apps/dashboard/pkg/apis/dashboard_manifest.go

```diff
@@ -11,8 +11,6 @@ import (
 	"github.com/grafana/grafana-app-sdk/app"
 )
 
-var ()
-
```

Comment:
This is not something I've done but its probably done by the go linter commit hook, can you verify @onprem . Thank you 🙏